openedx/core/djangoapps/programs/tasks/v1/tasks.py

```diff
@@ -520,3 +520,41 @@ def revoke_program_certificates(self, username, course_key):
         LOGGER.info(u'There is no program certificates for user %s to revoke', username)
 
     LOGGER.info(u'Successfully completed the task revoke_program_certificates for username %s', username)
+
+
+@task(bind=True, ignore_result=True, routing_key=PROGRAM_CERTIFICATES_ROUTING_KEY)
```

Comment:
any particular reason for ignoring result?

Reply:
kept in accordance to the rest of the tasks present in this app.